src/openrct2-cli/CMakeLists.txt

```diff
@@ -16,5 +16,5 @@ file(GLOB_RECURSE OPENRCT2_CLI_SOURCES
 add_executable(${PROJECT_NAME} ${OPENRCT2_CLI_SOURCES})
 target_include_directories(${PROJECT_NAME} PRIVATE "${CMAKE_CURRENT_LIST_DIR}/..")
 ipo_set_target_properties(${PROJECT_NAME})
-target_link_libraries(${PROJECT_NAME} libopenrct2 Threads::Threads)
+target_link_libraries(${PROJECT_NAME} ${ICU_LIBRARIES} libopenrct2 Threads::Threads)
```

Comment:
It's also required for `openrct2`. In fact, it's required by `libopenrct2`.

Reply:
It looks like ICU is already included by libopenrct2 as part of src/openrct2/CMakeLists.txt Line 139. If libopenrct2 is being built statically, is it expected that the linker will pick up ICU from libopenrct2?

Reply:
The line you mentioned tries linking `${ICU_STATIC_LIBRARIES}`, which is empty and that is the root cause.

Reply:
unlike `${ICU_LIBRARIES}`, used with `STATIC=off`